website/sidebars.json

```diff
@@ -45,6 +45,17 @@
         "type": "subcategory",
         "label": "Angular Testing Library",
         "ids": ["angular-testing-library/intro"]
+      },
+      {
+        "type": "subcategory",
+        "label": "Reason Testing Libraries",
```

Comment:
Even though there are technically separate wrappers for dom- and react-, can we use singular (library) in the heading? Or would that be confusing?

Reply:
Or even rename all the wrapper headings to just "React", "Reason", etc.

Reply:
Sure. I could do Reason Testing Library and Reason-React Testing Library